server/views/pages/bookings/future.njk

```diff
@@ -1,7 +1,10 @@
 {% extends "partials/layout.njk" %}
+{%- from "govuk/components/tag/macro.njk" import govukTag -%}
 
 {% set activeLink = 'bookings-link' %}
 {% set pageTitle = "Bookings" %}
+{% set referenceType = "Booking reference" %}
```

Comment:
```suggestion
{% set referenceType = "Request reference" if visit.visitSubStatus == "REQUESTED" else "Booking reference" %}
```

If you handle the two cases here you don't need to override `referenceType` in line 28.